docker/docker-compose.yml

```diff
@@ -0,0 +1,70 @@
+# You can remove version: '3.8' if you are using Docker Compose v2 or later
+# but it's not an issue if you're using v2
+version: '3.8'
+
+services:
+  bedrock-node0:
+    build:
+      context: ..
+      dockerfile: docker/Dockerfile
+    container_name: bedrock-node0
+    ports:
+      - "8888:8888"
+      - "9000:9000"
+    networks:
+      bedrock-cluster:
+        aliases:
+          - node0
+    volumes:
+      - bedrock-db-node0:/var/db
+    environment:
+      - NODE_NAME=node0
+      - PRIORITY=100
+      - PEER_LIST=node1:9000,node2:9000
+
+  bedrock-node1:
+    build:
+      context: ..
+      dockerfile: docker/Dockerfile
+    container_name: bedrock-node1
+    ports:
+      - "8887:8888"
+      - "9001:9000"
+    networks:
+      bedrock-cluster:
+        aliases:
+          - node1
+    volumes:
+      - bedrock-db-node1:/var/db
+    environment:
+      - NODE_NAME=node1
+      - PRIORITY=101
+      - PEER_LIST=node0:9000,node2:9000
+
+  bedrock-node2:
+    build:
+      context: ..
+      dockerfile: docker/Dockerfile
+    container_name: bedrock-node2
+    ports:
+      - "8889:8888"
```

Comment:
should this be `8889:8889`?

Reply:
yes @coleaeason  it should, it mean open the port 8887 into the host and map it with the 8888 into the container, like that people can make a peer list outside of the container with :8887, 8888, 8889